AppKit/CPTextView/CPTextView.j

```diff
@@ -1970,10 +1970,10 @@ var kDelegateRespondsTo_textShouldBeginEditing
 
         if (!numberOfGlyphs)
         {
-            var font = [_typingAttributes objectForKey:CPFontAttributeName];
+            var font = [_typingAttributes objectForKey:CPFontAttributeName] || [self font];
```

Comment:
In what circumstances _typingAttributes don't have (or lost) their CPFontAttributeName ? Is it something we should allow ?